README.md

```diff
@@ -1,12 +1,12 @@
 # webpackDemo
 
-> This project was shows webpack bunddle. Using html web pack plugin, html template, css and local dev server.
+> This project is a simple HTML list of To Do tasks. Built using webpack and served by a webpack server.
 
 
 ## Built With
 
 - Html, Js, Css.
-- webpack.
+- Webpack.
 - Linters.
 
```

Comment:
- [OPTIONAL] Please it will be nice to add a `setup section` with `explicit steps` to clone and run your app locally, as this will be of help to none technical users who would like to know how your app works.